### PR TITLE
[test] Squelch some console logging

### PR DESCRIPTION
**Pull request by STRML**
Just a minor nitpick, this kills some of the console logging in the test suite.

### CODE REVIEW

File: test/primus.test.js [hunk 1]
```diff
@@ -680,7 +686,8 @@ describe('Primus', function () {
 
     it('applies the options to the Primus server', function (done) {
       var port = common.port
-        , primus = Primus.createServer({ port: port, transformer: 'engine.io' });
+        , primus = Primus.createServer(
```
Comment (lpinca) [inline test/primus.test.js]: Minor style nit: get rid of the `port` variable and use one line per option in the options object.

File: test/primus.test.js [hunk 2]
```diff
@@ -766,3 +773,9 @@ describe('Primus', function () {
     });
   });
 });
+
+function stub(obj, prop) {
+  var stubbed = obj[prop];
```
Comment (lpinca) [inline test/primus.test.js]: `const`.

Comment (STRML) [inline test/primus.test.js]: Was just copying the style of the rest of the tests; we're going ES6 for all new code?

Comment (lpinca) [inline test/primus.test.js]: Yes, we dropped support for Node.js < 4 but haven't updated the code base yet.

File: test/primus.test.js [hunk 3]
```diff
@@ -225,22 +225,28 @@ describe('Primus', function () {
   });
 
   it('throws a human readable error for an unsupported transformer', function () {
+    var restoreConsole = stub(console, 'error');
```
Comment (lpinca) [inline test/primus.test.js]: Could be `const` as well.

Comment (lpinca) [inline test/primus.test.js]: It can be updated in another PR, no problem.
